applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -43,6 +43,11 @@ The USB stack is selected by enabling one of the following Kconfig choice option
   The HID boot protocol integration is not yet fully tested and may not work properly.
   The secondary image slot background erase in :ref:`nrf_desktop_dfu` may cause missing USB HID subscriptions after a USB cable is attached.
 
+The USB next stack is the only stack that supports the USB High-Speed.
+
+The only supported USB stack for the nRF54H20 is the USB next stack.
+The nRF54H20 supports the USB High-Speed.
```

Comment:
I would suggest to move this information under `CONFIG_DESKTOP_USB_STACK_NEXT ` bullet point,
Maybe we can also simplify it a bit: `This is the only USB stack that supports USB High-Speed and nRF54H20.`